posts/reflections/Lessons_from_building_Krets.md

```diff
@@ -0,0 +1,70 @@
+# Lessons from building Krets
+## Trying to create a profitable project alone
+
+I've spent most of my time between summer and christmas 2020 working on [Krets](https://krets.app). The project was 
+my first real attempt at creating a commercially viable service on my own. While the project isn't dead, I've now started focusing on other projects for a while. 
+
+This post covers some of the things I I've learned while working on Krets. 
+
+## Don't make me think! (or, frankly, do anything)
+Most people, i.e. potential customers, really don't like having to figure out how to use your software.
+I thought knew this after reading [Steve Krugh's _Don't make me think_](https://en.wikipedia.org/wiki/Don%27t_Make_Me_Think) years ago. 
+I didn't. 
+
+Krets was primarily targeted at businesses, meaning that I've been talking a lot to owners of small businesses. 
+They have a lot on their mind already. Everything not core to their business is a distraction unless 
+the rewards for them materialize themselves quickly. This means: minimal setup and minimal distruption 
+to their existing flow is essential. 
+
+I've experienced potential customers don't even want to spend the time to create an account, but would 
+rather have me do it. These were otherwize excited customers who showed initiative. This 
+This happened on several occations. 
+
+So the lesson is: People are not going to spend time on you or your project. 
+Even if they see value in what you're offering, they don't care until the value is actually there. 
+They don't find the project in itself interesting, like you do. 
+
+This was (still is, if I'm being honest) hard to internalize because I am 
+the oposite. I am curious to learn about sotware and play around with every setting I can find. 
+I guess that's why I ended up writing software.
+
+Also, if something else has to be handled (like idk, [a pandemic](https://en.wikipedia.org/wiki/COVID-19_pandemic)) 
+you're the first thing going off their todo-list. Understandably so :)
```

Comment:
: (

Reply:
Er nok mange som har slitt mer med det der enn meg for å si det forsiktig 😅